packages/react-core/src/next/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import styles from '@patternfly/react-styles/css/components/Dropdown/dropdown';
+import { css } from '@patternfly/react-styles';
+import { Menu, MenuContent, MenuProps } from '../../../components/Menu';
+import { Popper } from '../../../helpers/Popper/Popper';
+
+export interface DropdownProps extends MenuProps {
+  /** Anything which can be rendered in a dropdown */
+  children?: React.ReactNode;
+  /** Classes applied to root element of dropdown */
+  className?: string;
+  /** Renderer for a custom dropdown toggle. Forwards a ref to the toggle. */
+  toggleRender?: (toggleRef: React.RefObject<any>) => React.ReactNode;
```

Comment:
I think this could be called `toggle` but I'm okay either way. Render feels a little redundant because the typing is a function.